src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
+@import './modules/card';
+@import './modules/header';
+@import './modules/main';
```

Comment:
Ensure that the imported SCSS modules ('card', 'header', 'main') exist in the specified path './modules/'. If any of these files are missing, the styles associated with them will not be applied.